keyboards/epomaker/tide75/info.json

```diff
@@ -0,0 +1,264 @@
+            {"matrix": [2, 14], "x": 224, "y": 26, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 38, "flags": 4},
+            {"matrix": [3, 1], "x": 16, "y": 38, "flags": 4},
+            {"matrix": [3, 2], "x": 32, "y": 38, "flags": 4},
+            {"matrix": [3, 3], "x": 48, "y": 38, "flags": 4},
+            {"matrix": [3, 4], "x": 64, "y": 38, "flags": 4},
+            {"matrix": [3, 5], "x": 80, "y": 38, "flags": 4},
+            {"matrix": [3, 6], "x": 96, "y": 38, "flags": 4},
+            {"matrix": [3, 7], "x": 112, "y": 38, "flags": 4},
+            {"matrix": [3, 8], "x": 128, "y": 38, "flags": 4},
+            {"matrix": [3, 9], "x": 144, "y": 38, "flags": 4},
+            {"matrix": [3, 10], "x": 160, "y": 38, "flags": 4},
+            {"matrix": [3, 11], "x": 176, "y": 38, "flags": 4},
+            {"matrix": [3, 12], "x": 192, "y": 38, "flags": 4},
+            {"matrix": [3, 13], "x": 208, "y": 38, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 38, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 51, "flags": 4},
+            {"matrix": [4, 1], "x": 16, "y": 51, "flags": 4},
+            {"matrix": [4, 2], "x": 32, "y": 51, "flags": 4},
+            {"matrix": [4, 3], "x": 48, "y": 51, "flags": 4},
+            {"matrix": [4, 4], "x": 64, "y": 51, "flags": 4},
+            {"matrix": [4, 5], "x": 80, "y": 51, "flags": 4},
+            {"matrix": [4, 6], "x": 96, "y": 51, "flags": 4},
+            {"matrix": [4, 7], "x": 112, "y": 51, "flags": 4},
+            {"matrix": [4, 8], "x": 128, "y": 51, "flags": 4},
+            {"matrix": [4, 9], "x": 144, "y": 51, "flags": 4},
+            {"matrix": [4, 10], "x": 160, "y": 51, "flags": 4},
+            {"matrix": [4, 11], "x": 176, "y": 51, "flags": 4},
+            {"matrix": [4, 13], "x": 208, "y": 51, "flags": 4},
+            {"matrix": [4, 14], "x": 224, "y": 51, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [4, 1], "x": 16, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 32, "y": 64, "flags": 4},
+            {"matrix": [4, 3], "x": 48, "y": 64, "flags": 4},
+            {"matrix": [4, 4], "x": 64, "y": 64, "flags": 4},
+            {"matrix": [4, 6], "x": 96, "y": 64, "flags": 4},
+            {"matrix": [4, 7], "x": 112, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 144, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 160, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 176, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 192, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 208, "y": 64, "flags": 4},
+            {"matrix": [4, 14], "x": 224, "y": 64, "flags": 4}
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "force_nkro": true,
+        "pid": "0xE471",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x342D"
```

Comment:
This looks to be Westberry's vendor ID.  Do you have permission from them to use this VID+PID?

Reply:
Yes, we have permission to use it. We applied to Westberry.